charts/drupal/templates/varnish-configmap-vcl.yaml

```diff
@@ -15,19 +15,7 @@ data:
     backend prod1_http1 {
       .host = "{{ .Release.Name }}-drupal";
       .port = "80";
-      .max_connections = 300;
-
-      .probe = {
-        .url       = "/robots.txt"; 
-        .interval  = 5s; # check the health of each backend every n seconds
-        .timeout   = 1s; # timing out after n seconds
```

Comment:
Let's increase this to 5s, in case of slow response of a single page, it might bring the whole backend offline. And there's only a single backend contact point (service) in this architecture.

Reply:
Done, in `charts/drupal/values.yaml` line 466